code/mob/living/life/disability.dm

```diff
@@ -37,12 +37,9 @@
 			if (A.irradiated)
 				//spatial interdictor: mitigate effect of radiation
 				//power expenditure is managed centrally by the interdictor
-				var/interdictor_influence = 0
-				for_by_tcl(IX, /obj/machinery/interdictor)
-					if (IX.radstorm_interdict(owner))
-						interdictor_influence = 1
-						break
-				if(!interdictor_influence)
+				if (owner.hasStatus("spatial_protection"))
+					return
```

Comment:
Returning here prevents the checks further below from happening. Could invert the hasStatus check with ! instead.

Reply:
Alright, done